packages/hsl-shared-components/src/Touchable/TouchableNative.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/primitives';
+import { TouchableWithoutFeedback } from 'react-native';
+import { View as ViewPrimitives } from 'react-primitives';
+import View from '../View';
+
+class TouchableNative extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      pressed: false,
+    };
+    this.handlePressIn = this.handlePressIn.bind(this);
+    this.handlePressOut = this.handlePressOut.bind(this);
+  }
+
+  handlePressIn() {
+    this.setState({
```

Comment:
Instead of setState setNativeProps should be used.
https://facebook.github.io/react-native/docs/direct-manipulation.html

Reply:
In this case it probably doens't work since children should be rerendered